### PR TITLE
Ria 2110

### PR DESCRIPTION
**Pull request by mkayad**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RIA-2110

### Change description ###

see ticket

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X ] No
```

### CODE REVIEW

File: src/main/resources/application.yaml [hunk 1]
```diff
@@ -347,8 +347,8 @@ prof.ref.data.path.org.users: ${PROF_REF_DATA_ORG_USERS_PATH:/refdata/external/v
 
 launchDarkly:
   sdkKey: ${LAUNCH_DARKLY_SDK_KEY:sdk-key}
-  connectionTimeout: 5
-  socketTimeout: 5
+  connectionTimeout: 5000
```
Comment (Lee365) [inline src/main/resources/application.yaml:350]: This file change needs to be reverted

**Review by Lee365 — CHANGES_REQUESTED** [review]

File: src/main/resources/application.yaml (hunk 1 above)
Comment (mkayad) [reply src/main/resources/application.yaml]: changed to normal timeout times

**Review by Lee365 — APPROVED** [review]
LGTM

**Review by Lee365 — CHANGES_REQUESTED** [review]

**Review by Lee365 — APPROVED** [review]
LGTM

File: src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/entities/OutOfCountryDecisionType.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+import static java.util.Arrays.stream;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+import java.util.Optional;
+
+public enum OutOfCountryDecisionType {
+
+    REFUSAL_OF_HUMAN_RIGHTS("refusalOfHumanRights", "A decision to refuse a human rights claim for entry clearance"),
+    REFUSAL_OF_PROTECTION("refusalOfProtection", "A decision to refuse a human rights or protection claim, or deprive you of British citizenship, where you can only apply after your client has left the country"),
```
Comment (Lee365) [inline src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/entities/OutOfCountryDecisionType.java:11]: ccd-def has this "refusalOfPA"
Either this has to be corrected or ccd-def

**Review by Lee365 — CHANGES_REQUESTED** [review]

File: src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/entities/OutOfCountryDecisionType.java (hunk 2 above)
Comment (Lee365) [reply src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/entities/OutOfCountryDecisionType.java:11]: ccd-def corrected

File: src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/HomeOfficeCaseNotificationsHandler.java [hunk 3]
```diff
@@ -79,7 +81,17 @@ public boolean canHandle(
...
 
-        AsylumCase asylumCaseWithHomeOfficeData = homeOfficeApi.call(callback);
+        AsylumCase asylumCaseWithHomeOfficeData =
+            callback
+                .getCaseDetails()
+                .getCaseData();
+
+
+        if (asylumCaseWithHomeOfficeData.read(APPELLANT_IN_UK, YesOrNo.class).map(
+            value -> value.equals(YesOrNo.YES)).orElse(false)) {
```
[the remark this answers is not in the document]
Comment (Lee365) [reply src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/HomeOfficeCaseNotificationsHandler.java]: changed

File: src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/HomeOfficeCaseValidateHandler.java [hunk 4]
```diff
@@ -71,66 +64,74 @@ public boolean canHandle(
...
-        if (contactPreference.isPresent()) {
-            asylumCaseWithHomeOfficeData.write(
-                CONTACT_PREFERENCE_DESCRIPTION, contactPreference.get().getDescription());
+        AsylumCase asylumCaseWithHomeOfficeData =
+            callback
+                .getCaseDetails()
+                .getCaseData();
+
+        if (asylumCaseWithHomeOfficeData.read(APPELLANT_IN_UK, YesOrNo.class).map(
+            value -> value.equals(YesOrNo.YES)).orElse(false)) {
```
[the remark this answers is not in the document]
Comment (Lee365) [reply src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/HomeOfficeCaseValidateHandler.java]: changed

**Review by Srihari-Nakkala — DISMISSED** [review]
LGTM

**Review by Lee365 — DISMISSED** [review]

**Review by Lee365 — APPROVED** [review]
LGTM

**Pull request merged**